README.md

```diff
@@ -39,12 +45,11 @@ func main() {
   }
 
   // Dump database to file
-  err := dumper.Dump()
-  if err != nil {
+  if err := dumper.Dump(); err != nil {
     fmt.Println("Error dumping:", err)
     return
   }
-  fmt.Printf("File is saved to %s", dumpFilenameFormat)
+  fmt.Printf("File is saved to %s\n", dumper.Out.(*os.File).Name())
```

Comment:
If we are casting to an underlying type can we check if it truly is a part of that interface?

```go
if file, ok := dumper.Out.(*os.File); ok {
  fmt.Println("File is saved to", file.Name())
}
```

Reply:
yep, `dumper.Out.(*os.File)` should be OK here, if not would be failed in previous dumper register, but I'll update according to your suggestion  if you think it must be checked.

Reply:
Correct. I just worry about people copying things directly out of the README. `doc.go` contains the same snippet as well. I'd be fine with either removing `doc.go` or updating it with the new README information

Reply:
You're right, just updated it.